salt/metalk8s/addons/ui/files/metalk8s-ui-deployment.yaml

```diff
@@ -27,7 +27,6 @@ spec:
       containers:
         - name: metalk8s-ui
           image: {{ build_image_name('metalk8s-ui', '0.2') }}
-          imagePullPolicy: Always
```

Comment:
Why not putting `IfNotPresent` instead as we do for some other images? Better be explicit than implicit I'd say.

Reply:
@TeddyAndrieux: it is not by default?

Reply:
It's the default value but better to be explicit like for other manifest

Reply:
ok